rpcs3/Emu/Cell/lv2/sys_net.cpp

```diff
@@ -2939,6 +2958,12 @@ error_code sys_net_bnet_socket(ppu_thread& ppu, s32 family, s32 type, s32 protoc
 		{
 			return -get_last_error(false);
 		}
+		u32 default_RCVBUF = (protocol==IPPROTO_TCP) ? 65535 : 9216;
```

Comment:
protocol is only relevant for SOCK_RAW sockets, you should distinguish through socket type, SOCK_DGRAM = udp, SOCK_STREAM = tcp.

Reply:
I'll go for that option (SOCK_STREAM)